util/daemonize.c

```diff
@@ -7,6 +7,41 @@
 
 #include "daemonize.h"
 
+/* Read one byte from ${fd}, looping upon EINTR.  Do not store that byte. */
+static int
+noeintr_readbyte(int fd)
```

Comment:
Since we have a `noeintr.c`, we shouldn't have functions named `noeintr_*` in other files.

Reply:
done